docarray/helper.py

```diff
@@ -495,3 +497,23 @@ def _get_array_info(da: 'DocumentArray'):
             )
 
     return is_homo, _nested_in, _nested_items, attr_counter, all_attrs_names
+
+
+def login(force: bool = False):
+    """Login to Hubble account.
+    :param force: If set to true, overwrite token and re-login.
+    """
+    hubble.login(force=force)
+
+
+def logout():
+    """Logout Hubble account."""
+    hubble.logout()
+
+
+def notebook_login(force: bool = False):
```

Comment:
wait because `hubble-sdk` is working on a feature to autodetect it from notebook

Reply:
yes aware of it